grails-app/services/org/pih/warehouse/core/ConfigService.groovy

```diff
@@ -0,0 +1,37 @@
+package org.pih.warehouse.core
+
+import grails.core.GrailsApplication
+import grails.util.Holders
+
+/**
+ * Handles fetching property values from the app configuration.
+ */
+class ConfigService {
```

Comment:
This was an attempt to unify our approach to fetching config.

It's based on best practices defined here: https://grails.org/blog/2016-08-31.html

It also makes mocking config in tests a lot easier.